build.gradle

```diff
@@ -12,9 +12,9 @@ plugins {
     id 'com.github.ben-manes.versions' version '0.17.0'
 }
 
-ext.developmentVersion = '0.22.0-SNAPSHOT'
+ext.developmentVersion = '0.31.0-RC1-SNAPSHOT'
```

Comment:
Has this version been selected to align with the OT version? If so, shouldn't it be RC2?

Reply:
I just wanted to skip Jaeger v0.22. Can go to 0.23, no particular reason to align with OT

Reply:
The problem is that version v0.22 is getting mixed up - if the aim is release 0.22 based on scopes (as discussed in #311) then it would be better to release this as 0.22.0-RC4?

Otherwise we should release 0.22 based on activespan soon, and produce a 0.23-RC1 for scope version.

Reply:
`0.22` is very mixed up: 

* `0.22.0-RC3`  `0.22.0-RC2` - OT API 0.30.0
*  `0.22.0-RC1`  - OT API 0.31.0-RC1

Let's keep 0.22. with  OT 0.30 and use 0.23 with OT 0.31. 0.22 doesn't  even have to be released (it was mentioned on the last call).

The problem is with people who updated to RC2-3, bump to the GA version should not be breaking.

Reply:
changed to 0.23. I am on board with not releasing official 0.22 at all.